docs/api/cozy-client.md

```diff
@@ -883,6 +885,23 @@ revocation and token refresh.
 If `oauth` options are passed, stackClient is an OAuthStackClient.
 
 **Kind**: instance method of [<code>CozyClient</code>](#CozyClient)  
+<a name="CozyClient+getInstanceOptions"></a>
+
+### cozyClient.getInstanceOptions() ⇒ <code>object</code>
+getInstanceOptions - Returns a set of instance options (such as domain or current app slug) that has been previously loaded
```

Comment:
When reading "a set", I am thinking of the Set object. What do you think of "Return instance options" ?

"that has been previously loaded": it is unclear from where and how, maybe add something like "Should be called after one of CozyClient::loadInstanceOptions method has been called" ?

Reply:
A throw if instance options is not defined could be helpful for the programmer ?